server/test/unit/server/db/test_manage.py

```diff
@@ -6,15 +6,16 @@
 import os
 
 from mock import call, inPy3k, MagicMock, patch
+from mongoengine.queryset import DoesNotExist
 
 from ... import base
 from pulp.common.compat import all, json
 from pulp.server.db import manage
 from pulp.server.db.migrate import models
 from pulp.server.db.model.migration_tracker import MigrationTracker
-from pulp.server.managers.migration_tracker import DoesNotExist, MigrationTrackerManager
 import pulp.plugins.types.database as types_db
 import migration_packages.a
+import migration_packages.b
```

Comment:
We cannot use migration_packages.a because they are used in other testcases. I needed ne package that would be imported but not included here https://github.com/ipanova/pulp/blob/migration-tracker/server/test/unit/server/db/test_manage.py#L46